plugins/Toolbox/src/CloudSync/RestartApplicationPresenter.py

```diff
@@ -15,10 +15,10 @@ def __init__(self, app: CuraApplication) -> None:
     def present(self) -> None:
         app_name = self._app.getApplicationDisplayName()
 
-        message = Message(self._i18n_catalog.i18nc(
-            "@info:generic",
-            "You need to quit and restart {} before changes have effect.", app_name
-        ))
+        message = Message(self._i18n_catalog.i18nc("@info:generic",
+                                                   "You need to quit and restart {} before changes have effect.",
+                                                   app_name),
+                          message_type = Message.MessageType.WARNING)
```

Comment:
I don't think that this is a warning. It's a notification / clarification. A warning indicates that something is wrong, right?

Reply:
change to neutral